usaspending_api/broker/management/commands/load_fsrs.py

```diff
@@ -66,13 +66,13 @@ def generate_unique_ids(row, award_type):
             # "CONT_AW_" + contract_agency_code + contract_idv_agency_code + contract_number + idv_reference_number
             return (
                 'CONT_AW_'
-                + (row['contract_agency_code'].replace('-', '') if row['contract_agency_code'] else '-NONE-')
+                + (row['contract_agency_code'].replace('-', '') if row['contract_agency_code'] else 'NONE')
```

Comment:
I'm confused by this, I see the dashes in the awards table. Example: `CONT_AW_97AS_-NONE-_07000198909DDLA70089C3135_-NONE-`

Reply:
Weird. I thought it matched more with the `NONE`s instead of `-NONE-`s. Double checking.

Reply:
Figured it out. Check out line `104` in `get_award`:

```
            # Find the award to attach this sub-contract to, using the generated unique ID:
            award = (
                Award.objects.annotate(
                    modified_generated_unique_award_id=Func(
                        F('generated_unique_award_id'), Value('-'), Value(''), function='replace'
                    )
                )
                .filter(
                    modified_generated_unique_award_id=self.generate_unique_ids(row, award_type),
                    latest_transaction_id__isnull=False,
                )
                .distinct()
                .order_by("-date_signed")
                .first()
            )
```
It matches the two ids by removing the `-` and comparing them to this ID with the `-NONE-`, causing a mismatch,.